tmt/steps/report/reportportal.py

```diff
@@ -0,0 +1,89 @@
+import dataclasses
+import io
+import os
+import zipfile
+from typing import List, Optional
+
+import click
+import junit_xml
+
+import tmt.steps.report
+import tmt.utils
+
+from . import junit
+
+
+@dataclasses.dataclass
+class ReportPortalData(tmt.steps.report.ReportStepData):
+    url: Optional[str] = None
+    token: Optional[str] = None
+    launch_name: Optional[str] = None
+
+
+@tmt.steps.provides_method("reportportal")
+class ReportPortal(tmt.steps.report.ReportPlugin):
+    """
+    Send results in JUnit format to Report Portal instance.
+    """
```

Comment:
Please, mention the supported variables in the docstring as well and add a new story to the [report step specification](https://github.com/teemtee/tmt/blob/main/spec/plans/report.fmf). Thanks.

Reply:
The docstring does not seem to be updated. This is the part which is displayed to user when `--help` option is provided on the command line. You can get some inspiration [here](https://github.com/teemtee/tmt/blob/main/tmt/steps/discover/fmf.py#L73).

Reply:
Covered in 017e4389.